pallets/phala/src/compute/basepool.rs

```diff
@@ -334,6 +364,12 @@ pub mod pallet {
 			Ok(guard)
 		}
 
+		/// The core procedure that contributes some stake to a pool
+		///
+		/// Before mint a new nft to the delegater, the function will try to merge all nfts in the pool-collection into on unify nft
```

Comment:
```suggestion
		/// Before minting a new nft to the delegator, the function will try to merge all nfts in the pool-collection into the unified nft
```

Reply:
Didn't post more comments to the docstrings below. Generally the description is good. Let's refine that with the cautions:

- Grammar: can be checked with [grammarly](https://www.grammarly.com/)
- Spacing, case, and punctuations
- Best practice of function comments by Google

Reply:
done